dbms/src/Storages/KVStore/KVStore.cpp

```diff
@@ -497,7 +497,8 @@ void KVStore::reportThreadAllocInfo(std::string_view v, ReportThreadAllocateInfo
 
 void KVStore::reportThreadAllocBatch(std::string_view v, ReportThreadAllocateInfoBatch data)
 {
-    joint_memory_allocation_map->reportThreadAllocBatchForKVStore(v, data);
+    // Initialize Context before RaftStoreProxyRunner, if it should be changed into non-static method.
+    JointThreadInfoJeallocMap::reportThreadAllocBatchForKVStore(v, data);
```

Comment:
I don't get it.

If KVStore is not inited, then it should be block by 
https://github.com/pingcap/tiflash/blob/23784449d406b46009ce7e63f7eea547ac76d621/dbms/src/Storages/KVStore/FFI/ProxyFFI.cpp#L1062-L1065

If KVStore is inited, then there is no code to reset `joint_memory_allocation_map`?

https://github.com/pingcap/tiflash/blob/23784449d406b46009ce7e63f7eea547ac76d621/dbms/src/Storages/KVStore/KVStore.cpp#L56-L76